rental.rb

```diff
@@ -0,0 +1,15 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
```

Comment:
Just as the previous reviewer has suggested, kindly add the book and date as attributes of the `initialize` method👍.

it should look this way😉

```suggestion
  def initialize(date, book, person)
```

Reply:
@Ibizugbe I dont understand what you mean